src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.styles.ts

```diff
@@ -20,6 +20,7 @@ const classNames = {
     dropdownTitleText: `${modelledPropertyBuilderClassPrefix}-dropdown-title-text`,
     loadingSpinnerContainer: `${modelledPropertyBuilderClassPrefix}-loading-spinner-container`,
     labelContainer: `${modelledPropertyBuilderClassPrefix}-label-container`,
+    clearButton: `${modelledPropertyBuilderClassPrefix}-clear-button`,
```

Comment:
nit: can we alphabetize the list. 😆 I'm a pain, I know.

Reply:
haha no prob